app/controllers/steps/claim_reason_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Steps
+  class ClaimReasonController < Steps::BaseStepController
```

Comment:
This is my fault, but think we are better off use @imtiazAhmd naming convestions for the steps (making this one ReasonForClaim instead

Reply:
should this be done for the entire app - all references to ClaimReason replaced with ReasonForClaim including filenames ?